lms/templates/vert_module.html

```diff
@@ -10,7 +10,11 @@ <h2 class="hd hd-2 unit-title">${unit_title}</h2>
 
 <div class="vert-mod">
 % for idx, item in enumerate(items):
-  <div class="vert vert-${idx}" data-id="${item['id']}">
+<div class="vert vert-${idx}" data-id="${item['id']}" \
+% if item['id'] in watched_completables:
+  data-completable \
```

Comment:
`data-completable-by-viewing` is long, but I find it less confusing than if one were trying to understand why other completable blocks don't seem to be completable.